templates/zerver/integrations/zoom.md

```diff
@@ -1,23 +1,37 @@
-Zulip supports using Zoom as its video call provider.
+# Zulip Zoom Integration
 
-### Using Zoom
+Make video calls from Zulip with Zoom!
 
-1. Select Zoom as the organization's [video call provider](/help/start-a-call#changing-your-organizations-video-call-provider).
+### Using Zoom in your Zulip organization
 
-1. Zulip's [call button](/help/start-a-call) will now create meetings
-   using Zoom.
+!!! warn ""
 
-The first time a user clicks the call button, they will be prompted to
-link their Zoom account with their Zulip account.
+    **Note**: If you are self-hosting, you will need to
+    [create a Zoom application][build-app].
 
-**Note**: If you are self-hosting, you will need to [create a Zoom
-    application](https://zulip.readthedocs.io/en/latest/production/video-calls.html#zoom)
-    in order to use this integration.
+To configure your Zulip organization to create Zoom meetings using Zulip's
+[call buttons](/help/start-a-call)
+(<i class="zulip-icon zulip-icon-voice-call"></i> /
+<i class="zulip-icon zulip-icon-video-call"></i>), set Zoom as your
+organization's [video call provider][change-provider].
 
-### Unlinking your Zoom account
+!!! tip ""
+
+    The default video call provider is [Jitsi Meet][Jitsi Meet doc].
+
+### Linking your Zoom account
 
-1. Log in to the [Zoom App Marketplace](https://marketplace.zoom.us/).
+The first time you click either of the [call buttons](/help/start-a-call)
+(<i class="zulip-icon zulip-icon-voice-call"></i> /
+<i class="zulip-icon zulip-icon-video-call"></i>) in Zulip, you will
+automatically be prompted to link your Zoom account with your Zulip account.
+
+### Unlinking your Zoom account
 
-1. Click **Manage** → **Installed Apps**.
+Log in to the [Zoom App Marketplace](https://marketplace.zoom.us/). Open
+**Manage**, select **Added Apps**, click the **Remove** button next to the
+Zulip app, and click **Confirm**.
```

Comment:
I think this is better as instructions...

1. Log in to the Zoom App Marketplace and select Manage.
2. Select Added Apps and click the Remove button next to the Zulip app.
3. Click Confirm.